Fluid/Ast/IncludeStatement.cs

```diff
@@ -45,6 +48,14 @@ public IncludeStatement(Expression path)
                 throw new FileNotFoundException(relativePath);
             }
 
+            if (AssignStatements != null)
+            {
+                foreach (var assignStatement in AssignStatements)
```

Comment:
I don't think these values should be visible after the include statement is executed. Probably necessary to call `context.EnterScope` and `content.LeaveScope` after.